### PR TITLE
Mg branch

### PR DESCRIPTION
**Pull request by mgratti**
- running time for dumper => note that now one has to always give 5 times, see option --time 
- possibility to split the dumper on several jobs, see option --splitfactord
- output of dumper on storage element now (for user=mratti)
- option to run only the postproduction and the dumper: run the same command for production and add the option --dodumperonly

### CODE REVIEW

**Comment by mgratti** [comment]
@amlyon: I merged the conflicts, such that I can also launch g+jet productions with the new threshold combinations. Now checking if my gjets run fine.
I merged with the idea of respecting the changes by both of us, but I would appreciate if you could have a look at the QCD workflow

Concerning the output of the dumper being copied to the storage element, for the moment I have enabled it only for me; I would tend to suggest that I enable it for you too, because Nina suggested that we don't run large jobs that continuously write on /work. Let me know.

**Comment by amlyon** [comment]
Hi, thanks a lot.

I will do some checks on the QCD then.

On my side, the jetmet validation made me realize that there was a 
mistreatment in the production of the QCD samples , I fixed it and am 
making the final checks now. I suggest that I first pull your proposed 
merge, and add my fix (which should not be in conflict with any with 
what you've done - but will be checked) to then open a new PR, if you 
agree.

Le 09.03.2020 17:38, Maria Giulia Ratti a écrit :
> @amlyon: I merged the conflicts, such that I can also launch g+jet
> productions with the new threshold combinations. Now checking if my
> gjets run fine.
> I merged with the idea of respecting the changes by both of us, but I
> would appreciate if you could have a look at the QCD workflow
> 
> Concerning the output of the dumper being copied to the storage
> element, for the moment I have enabled it only for me; I would tend to
> suggest that I enable it for you too, because Nina suggested that we
> don't run large jobs that continuously write on /work. Let me know.

**Comment by mgratti** [comment]
ok, sounds good to me. this means that you have to merge my PR, right?
if in the meantime I find some troubles with what I have done, will push with a new pull request then.

**Comment by amlyon** [comment]
Yes I first have to merge your PR, am going through it now.

Le 09.03.2020 17:56, Maria Giulia Ratti a écrit :
> ok, sounds good to me. this means that you have to merge my PR, right?
> if in the meantime I find some troubles with what I have done, will
> push with a new pull request then.
>

File: Dumpers/test/ECALproductionHelper/prodHelper.py [hunk 1]
```diff
@@ -50,12 +50,14 @@ def getOptions():
   parser.add_argument('--domedium', dest='domedium', help='set 2 days as wall clock time instead of 1 day', action='store_true', default=False)
   parser.add_argument('--dolong', dest='dolong', help='set 3 days as wall clock time instead of 1 day', action='store_true', default=False)
   parser.add_argument('--docustomtime', dest='docustomtime', help='set custom time', action='store_true', default=False)
-  parser.add_argument('--time', type=str, dest='time', help='requires docustomtime, allowed time for each job of each step in hours, example 01,02,01 for 1 hour for step1, 2 for step2, 2 for step3', default='02,02,05')
+  parser.add_argument('--time', type=str, dest='time', help='requires docustomtime, allowed time for each job of each step in hours, including dumper, example 01,02,01,01,04 for 1h for step1, 2h for step2, 1h for step3, 1h for step4, 4h for dumper', default='02,02,05,05')
```
Comment (amlyon) [inline Dumpers/test/ECALproductionHelper/prodHelper.py:53]: @mgratti 

May I ask you why 5 times are needed now? One usually does not run the step4 and the dumper for the same sample

(Also, there are only 4 times in the default)

**Comment by mgratti** [comment]
the default is a mistake!
the 5 times is one time for each step, but it’s true that one cannot run the dumper on top of step4

**Comment by amlyon** [comment]
@mgratti I am merging now, and proceeding to the QCD checks

**Pull request merged**